docs/release-notes.md

```diff
@@ -16,9 +16,12 @@ nav_order: 9
 ### Changes
 
 - Require Go 1.20+
+- The Dracut module now installs partx
 
 ### Bug fixes
 
+- Force partition table update after repartitioning, even if one partition is already mounted
```

Comment:
Minor: I personally think this is a new feature, not a bug fix (no assurances were previously made that this should've worked). E.g.

> Support partitioning disk with mounted partitions